ModelViewer/ModelViewerRun/ModelViewerRun.vcxproj

```diff
@@ -66,7 +66,7 @@
     </Link>
   </ItemDefinitionGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Debug|x64'" Label="QtSettings">
-    <QtInstall>msvc2019_64</QtInstall>
```

Comment:
Same comment as above, let's not commit this change because everyone else has their Qt named "msvc2019_64"